robot/rospackages/src/mcu_control/scripts/PdsNode.py

```diff
@@ -241,6 +245,12 @@ def publish_pds_data(message):
         return
     return
 
+def checkTempErrorCase(x):
```

Comment:
You should maybe document this function to explain why it needs to be there. So some guy in the future that didnt know about the 999.9 meme popup can understand